### PR TITLE
Dev anna serova tbkd

### PR DESCRIPTION
**Pull request by AnnaSerova**
#51

### CODE REVIEW

File: expected_conversions.view.lkml [hunk 1]
```diff
@@ -0,0 +1,234 @@
+view: account_avg_cpa {
```
Comment (jdavidson) [inline expected_conversions.view.lkml]: Can you move this to reuse the pattern in account_quarter_stats and pull in all of the sum measures and use extends?

It also will need some sort of date to use.

File: expected_conversions.view.lkml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+  dimension: total_conversions_dim {
+    description: "Total conversions."
+    value_format: "#,##0"
+    type: number
+    sql: ${TABLE}.total_conversions ;;
+    hidden: yes
+  }
+
+  measure: total_cost {
```
Comment (jdavidson) [inline expected_conversions.view.lkml]: These measures won’t be necessary to redefine with the extends

File: expected_conversions.view.lkml [hunk 3]
```diff
@@ -0,0 +1,234 @@
+view: account_avg_cpa {
+  derived_table: {
+    explore_source: master_stats {
+      column: external_customer_id {}
+      column: total_cost_usd {}
```
Comment (jdavidson) [inline expected_conversions.view.lkml]: We should also be sure to pull both total_cost and total_cost_usd for an foreign currencies

File: expected_conversions.view.lkml (hunk 1 above)
Comment (AnnaSerova) [reply expected_conversions.view.lkml]: updated it to use the same pattern and added a date filter

Comment (AnnaSerova) [reply expected_conversions.view.lkml]: ok

**Pull request merged**